src/main/java/lesson02/PalindromeChecker.java

```diff
@@ -0,0 +1,31 @@
+package lesson02;
+/*
+Task 4 (PalindromeChecker)
+Определить, является ли строка палиндромом. Палиндром - строка, которая с двух сторон читается одинаково.
+Создавать новые строки запрещено.
+Ограничения: нет.
+boolean isPalindrome(String s)
+Входные аргументы: String s
+Тип результата: boolean
+*/
+
+public class PalindromeChecker {
+    public static void main (String[] args) {
+        System.out.println(isPalindromeChecking("12345 6 7 6 54321"));
+    }
+
+    private static boolean isPalindromeChecking (String inputString){
```

Comment:
Required and actual method names aren't the same.

Reply:
Method and boolean variable renamed